doc/html/src/index.html

```diff
@@ -26,19 +26,19 @@
 CFDLab in March 2002.  Major
 contributions have come from developers at the Technische
 Universit&auml;t Hamburg-Harburg
-<a href="http://www.mub.tu-harburg.de">Institute of Modelling and Computation</a>,
+<a href="https://www.tuhh.de/alt/mub/institute.html">Institute of Modelling and Computation</a>,
 and recent contributions have been made by CFDLab associates at
-<a href="http://pecos.ices.utexas.edu/">the PECOS Center</a> at <a
-href="http://www.utexas.edu/">UT-Austin</a>,
-<a href="https://inlportal.inl.gov/portal/server.pt?open=514&objID=1269&mode=2&featurestory=DA_574924">the
+<a href="https://pecos.ices.utexas.edu/">the PECOS Center</a> at <a
+href="https://www.utexas.edu/">UT-Austin</a>,
+<a href="https://www.inl.gov/article/advanced-modeling-simulation/">the
 Computational Frameworks Group</a> at
-<a href="http://www.inl.gov/">Idaho National Laboratory</a>,
-<a href="http://www.nasa.gov/">NASA</a> <a href="http://www.nasa.gov/centers/johnson/home/index.html">Lyndon B. Johnson Space Center</a>,
-and <a href="http://augustine.mit.edu">MIT</a>.
```

Comment:
The defunct augustine.mit.edu link originally referred to Prof. Anthony Patera's research group at MIT, looks like it may have moved to http://pateragroup.mit.edu/ but there is currently no content there.